README.md

```diff
@@ -1,3 +1,3 @@
 ## You can see all projects in action here
 
-[Projects](https://www.vanillajavascriptprojects.com/)
+Click on the Link - [Projects](https://www.vanillajavascriptprojects.com/)
```

Comment:
Click On The This Link - [ Projects ] ( https://www.vanillajavascriptprojects.com/ )